src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -116,17 +125,22 @@ export default class CardanoSignTransaction extends AbstractMethod {
             });
         }
 
+        let auxiliaryData;
+        if (payload.auxiliaryData) {
+            auxiliaryData = transformAuxiliaryData(payload.auxiliaryData);
+        }
+
         this.params = {
             inputs,
             outputs,
             fee: payload.fee,
             ttl: payload.ttl,
             certificates,
             withdrawals,
-            metadata: payload.metadata,
             validity_interval_start: payload.validityIntervalStart,
             protocol_magic: payload.protocolMagic,
             network_id: payload.networkId,
+            auxiliary_data: auxiliaryData,
```

Comment:
I'd prefer the ordering consistent with tx body serialization (so no protocol magic in between tx body parts).

Reply:
Moved - https://github.com/vacuumlabs/connect/pull/12/commits/cda61468472cd09c479f875e8b2d7b80923ad4e9.